src/tasks/nli_probing.py

```diff
@@ -7,123 +7,95 @@
 from .registry import register_task
 from ..utils.data_loaders import load_tsv, process_sentence
 
+@register_task('nps', rel_path='nps/')
+class NPSTask(PairClassificationTask):
+
+    def __init__(self, path, max_seq_len, name, **kw):
+        super(NPSTask, self).__init__(name, n_classes=3, **kw)
+        self.load_data(path, max_seq_len)
+        self.sentences = self.val_data_text[0] + self.val_data_text[1]
+
+    def load_data(self, path, max_seq_len):
+        targ_map = {'neutral': 0, 'entailment': 1, 'contradiction': 2}
+        prob_data = load_tsv(self._tokenizer_name, os.path.join(path, 'dev.tsv'), max_seq_len,
+                            s1_idx=0, s2_idx=1, label_idx=2, label_fn=targ_map.__getitem__, skip_rows=0)
+        self.train_data_text = self.val_data_text = self.test_data_text = prob_data
+        log.info("\tFinished loading NP/S data.")
+
 
 @register_task('nli-prob', rel_path='NLI-Prob/')
 class NLITypeProbingTask(PairClassificationTask):
```

Comment:
Why all the structural changes here?

Reply:
This could be odd diff rendering, but I find it very hard to follow this...

Reply:
The move makes sense—I could just use more help figuring out what the cleanup is.